### ISSUE TITLE
[Adaptive Cards - Designer] - Luminosity ratio of the focus indicator on the menu items present inside the 'Select host app:' and 'Target version:' combo box with respect to its adjacent background color is 1.60:1, is less than 3:1.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9323]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Tool: Color Contrast analyzer -> Launch through Accessibility Insights for Windows Tool

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to the 'Designer' control present on the header section of the page using tab key and activate using enter key.
3. Navigate to the 'Select host app:' and 'Target version:' combo box using tab key and activate using enter key.
4. Now navigate to the menu items using down arrow key.
5. Launch Accessibility Insights for Windows tool and open Color contrast analyzer and select the foreground and adjacent background color of the focus indicator on the menu items.
6. Verify that, luminosity ratio of the focus indicator on the menu items present inside the 'Select host app:' and 'Target version:' combo box with respect to its adjacent background color is less than minimum required ratio i.e., 3:1 or not.

**Actual Result:**
Luminosity ratio of the focus indicator on the menu items present inside the 'Select host app:' and 'Target version:' combo box with respect to its adjacent background color is 1.60:1, which is less than minimum required ratio i.e., 3:1.

**Reproducibility:**
Same issue replicates for the menu items present inside the 'Add an action' combo box control. **Refer Attachment:** Luminosity_'Add an action'.

**Expected Result:**
Luminosity ratio of the focus indicator on the menu items present inside the 'Select host app:' and 'Target version:' combo box with respect to its adjacent background color should be greater than or equal to the minimum required ratio i.e., 3:1.

**User Experience:**
Low vision users will be impacted and might not identify the currently focused element on the page if luminosity ratio of the focus indicator is less than minimum required 3:1 on the  menu items.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/fe108890-185e-4ff5-a8e1-3d4b84559b69)

**Luminosity_'Add an action':**
![Image](https://github.com/user-attachments/assets/07b92e03-814d-403c-8089-8d8f0ca26b57)

### Card JSON

```JSON
NA
```
